packages/typescript/src/create-package.ts

```diff
@@ -16,9 +16,13 @@ export interface PackageDescriptor {
   [path: string]: ModuleSymbolsDescriptor;
 }
 
+export type NamedModuleDescriptor =
+  | string
+  | { name: string; staticMembers?: Array<string> }; // TODO: array of strings sufficient?
```

Comment:
It is not, because static members may have their own static members, e.g. `someLib.foo.bar`.

Reply:
Let's see... The package defines an exported module "./server/index.js" with a named export of "server" (`SymbolFlags.StaticMemberContainer`), which has a static member symbol "nested" (`SymbolFlags.StaticMember | SymbolFlags.StaticMemberContainer`), which has a static member symbol "nestedHandler" (`SymbolFlags.StaticMember`)

Then yes I would expect `mcpSdk["./server/index.js"].server.nested.nestedHandler` to vend a refkey for the `nestedHandler` symbol, and when resolving that refkey, to get the correct reference to it (which should hopefully work without any changes to referencing code which should already support this logic).